bot/interact.go

```diff
@@ -50,17 +51,32 @@ func interactAPIHandler(botContext cmd.BotContext) http.HandlerFunc {
 				dialog := generateReportMessageDialog()
 				dialog.State = slackx.LinkToMessage(message.Channel.ID, message.MessageTs) // persist the message link across submission
 				if err := botContext.Client.OpenDialog(message.TriggerID, dialog); err != nil {
-					log.Println(err.Error())
+					log.Println(err)
 				}
 			case "delete_job_post":
 				modal := generateDeleteJobPostModal()
 				modal.PrivateMetadata = fmt.Sprintf("%s|%s|%s", message.Channel.ID, message.Message.Text, message.MessageTs) // persist the message channel, text, and ts across submission
 
 				if resp, err := botContext.Client.OpenView(message.TriggerID, modal); err != nil {
-					log.Println(err.Error())
+					log.Println(err)
 					log.Println(strings.Join(resp.ResponseMetadata.Messages, "\n"))
 					log.Println(strings.Join(resp.ResponseMetadata.Warnings, "\n"))
 				}
+			case "delete_thread":
+				if !botContext.IsStaff(message.User.ID) {
+					log.Println("this action is only allowed to Staff members")
```

Comment:
maybe we could log the user id so we can know who's trying to cheat

Reply:
Done! I'm now also showing a modal to the user that looks like this: 
<img width="517" alt="2022-11-29 at 21 26 30@2x" src="https://user-images.githubusercontent.com/1083296/204651316-75be0f63-cb47-42a7-b141-4e01a4e839ec.png">